src/component/1d/Viewer1D.tsx

```diff
@@ -387,6 +381,16 @@ function Viewer1D({ emptyText = undefined }: Viewer1DProps) {
                     </MouseTracker>
                   </BrushTracker>
                 )}
+              {brushDataRef.current && (
```

Comment:
I haven't looked into it, but something is wrong here. You should never use a ref's value during rendering, because changing the ref doesn't trigger a rerender.

Reply:
fixed, I used a state.